README.md

```diff
@@ -28,12 +28,48 @@ When the reader has completed this Code Pattern, they will understand how to:
 * [Flask](http://flask.pocoo.org/): is a micro web framework.
 * [ngrok](http://ngrok.com/): allows you to open an HTTPS tunnel into your REST API.
 
-## Prerequisites:
+## Featured technologies
+
+* [Artificial Intelligence](https://medium.com/ibm-data-science-experience): Artificial intelligence can be applied to disparate solution spaces to deliver disruptive technologies.
```

Comment:
Should this link to the official product page? https://www.ibm.com/products/data-science-experience

Reply:
Oh. I didn't realize that - I copied it from https://github.com/IBM/powerai-counting-cars

Should I change it to the official?